src/app/commands/cmd_sprite_size.cpp

```diff
@@ -199,9 +222,23 @@ class SpriteSizeWindow : public app::gen::SpriteSize {
     method()->addItem("Nearest-neighbor");
     method()->addItem("Bilinear");
     method()->addItem("RotSprite");
-    method()->setSelectedItemIndex(
-      get_config_int("SpriteSize", "Method",
-                     doc::algorithm::RESIZE_METHOD_NEAREST_NEIGHBOR));
+    int resize_method;
+    if (params.method.isSet()) {
+      if (params.method() == doc::algorithm::RESIZE_METHOD_BILINEAR)
+        resize_method = 1;
```

Comment:
There is no need to do this manual conversion for each case between the `doc::algorithm::ResizeMethod` <-> `int`. We can just use:
```c++
resize_method = (int)params.method();
```
because we already were making the assumption that the integer value matches the value of the enum on  `get_config_int()`.

Reply:
Got it!